pkg/scheduler/reconciliation/filter.go

```diff
@@ -2,11 +2,28 @@ package reconciliation
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/kyma-incubator/reconciler/pkg/db"
 	"github.com/kyma-incubator/reconciler/pkg/model"
 )
 
+type WithCreationDateBefore struct {
+	Time time.Time
+}
+
+func (wd *WithCreationDateBefore) FilterByQuery(q *db.Select) error {
+	q.WhereRaw("created<$1", wd.Time.Format("2021-11-26 00:00:00.000"))
+	return nil
```

Comment:
Can we double check that it works with SQLite and Postgres? Not sure that both behave equal when processing timestamps.

Reply:
Thank you for this question. I've tested this on the `SQLitez and the `PostgreSQL` on my local machine and the spell is working for both DBs.

example spell:
```
SELECT * FROM scheduler_reconciliations WHERE created<'2021-11-03 23:23:01';
```